openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -222,16 +229,20 @@ TCN_IMPLEMENT_CALL(jlong, SSLContext, make)(TCN_STDARGS, jint protocol, jint mod
         goto cleanup;
     }
 
-    TCN_THROW_IF_ERR(apr_pool_create(&p, tcn_global_pool), p);
-
-    if ((c = apr_pcalloc(p, sizeof(tcn_ssl_ctxt_t))) == NULL) {
-        tcn_ThrowAPRException(e, apr_get_os_error());
+    if ((c = OPENSSL_malloc(sizeof(tcn_ssl_ctxt_t))) == NULL) {
```

Comment:
consider just using `calloc` instead of `OPENSSL_malloc`. IIUC OpenSSL is not aware of this variable and therefore we don't need to use OpenSSL allocation.